.github/workflows/docker.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         id: docker_meta
         uses: docker/metadata-action@v5
         with:
-          images: ghcr.io/pterodactyl/wings
+          images: ghcr.io/${{ github.repository }}
```

Comment:
I was about to make a PR with this change because my fork would not push to the correct repo with this "pterodactyl/wings" hardcoded, very nice!